art-cluster/pipelines/config/argocd/project/art-cd/quay-doomsday-backup/base/doomsday-pipeline-task.yaml

```diff
@@ -10,6 +10,9 @@ spec:
     - description: 'Full version, eg: 4.15.5'
       name: version
       type: string
+    - description: 'URL of the PipelineRun'
```

Comment:
Passing the PipelineRun URL to the job does not modify its functionality. I do not see any case where we have to pass a different pipelinerun URL than which can be automatically created. Which means that it need not be a param here.